docs/source/contribution/technical_steering_committee.md

```diff
@@ -36,18 +39,38 @@ On this page we describe:
 ## Requirements to become a maintainer
 
 Just contributing does not make you a maintainer; you need to demonstrate commitment to Kedro's long-term success by
-working with existing maintainers for a period of time.
+working with existing maintainers for at least one year.
 
-We look for commitment markers who can do the following:
+We look for commitment markers who can do at least some of the following:
 
 - Write high-quality code and collaborate with the team and community
+- Make substantial improvements to our documentation, either in terms of content or information architecture
+- Bring awareness of industry trends to the team, connect the project to complementary initiatives, help steer the direction
+- Give talks about the project at conferences and online events, write blog posts, or other forms of advocacy
+- Improve the aesthetics and usability of the project, including its documentation, website, and user interfaces (graphical and textual)
 - Understand the project's code base and internals
+- Develop valuable plugins that complement the functionality of Kedro
 - Make pull requests from our backlog or roadmap; maintainers need to work towards a common goal
 - Learn how the team works, including processes for testing, quality standards and code review
 - Show evidence of already having started pull requests and code reviews under the guidance of maintainers; including asking
```

Comment:
This is more a contributor -> committer thing

Reply:
Clarified in the new wording